confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -259,10 +264,10 @@ def get_by_id(self, schema_id):
 
         result, code = self._send_request(url)
         if code == 404:
-            log.error("Schema not found:" + str(code))
+            log.error("Schema not found. Error code: {}".format(code))
             return None
-        elif not (code >= 200 and code <= 299):
-            log.error("Unable to get schema for the specific ID:" + str(code))
+        elif not (200 <= code <= 299):
+            log.error("Unable to get schema with an ID of {}. Error code: {}".format(schema_id, code))
```

Comment:
Is there a reason why the wording changed here?

Reply:
I think "with specific ID {}" is grammatically fine. I was just curious. No reason to change